samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -66,12 +69,12 @@
 
   private ExecutorService executorService;
 
-  private volatile SamzaContainer container = null;
+  @VisibleForTesting
+  volatile SamzaContainer container = null;
   private volatile Throwable containerException = null;
+  private volatile Future<?> scheduledSamzaContainerFuture = null;
 
-  // Latch used to synchronize between the JobCoordinator thread and the container thread, when the container is
-  // stopped due to re-balancing
-  /* package private */volatile CountDownLatch jcContainerShutdownLatch;
+  /* package private */volatile CountDownLatch containerShutdownLatch;
```

Comment:
Don't need to add package private, it's implied. Also, why is this package private?

Reply:
Just to clarify this was not introduced as a part of this change. 
- Removed the comment as per your suggestion.
- It is set explicitly in StreamProcessor tests.